### PR TITLE
feat(ARC-1279): Show blade if long description + ARC-1278 (added alert)

### PR DESCRIPTION
**Pull request by WardVerc1**
image
image

### CODE REVIEW

File: src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	const { tText } = useTranslation();
+	const maxLength = 500;
+	const isDescriptionLong = description.length > maxLength;
+	const shortenedDescription = description.substring(0, maxLength);
+	const [isBladeOpen, setIsBladeOpen] = useState(false);
+
+	return (
+		<>
+			<p className="u-pb-24 u-line-height-1-4 u-font-size-14">
+				<TextWithNewLines text={isDescriptionLong ? shortenedDescription : description} />
```
Comment (silkederudder-shd) [inline src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx:25]:
```suggestion
				<TextWithNewLines text={desc} />
```

File: src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+				)}
+			</p>
+			<Blade
+				className={clsx(
+					styles['c-metadatadescription'],
+					'u-pb-24 u-line-height-1-4 u-font-size-14'
+				)}
+				isOpen={isBladeOpen}
+				onClose={() => setIsBladeOpen(false)}
+				renderTitle={() => (
```
Comment (silkederudder-shd) [inline src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx:39]: Maybe move this fn in a separate render function e.g `renderBladeTitle()`

File: src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	const isDescriptionLong = description.length > maxLength;
+	const shortenedDescription = description.substring(0, maxLength);
+	const [isBladeOpen, setIsBladeOpen] = useState(false);
+
+	return (
+		<>
+			<p className="u-pb-24 u-line-height-1-4 u-font-size-14">
+				<TextWithNewLines text={isDescriptionLong ? shortenedDescription : description} />
+				{isDescriptionLong && (
+					<u style={{ cursor: 'pointer' }} onClick={() => setIsBladeOpen(true)}>
```
Comment (silkederudder-shd) [inline src/modules/shared/components/MetaDataDescription/MetaDataDescription.tsx:27]: Not a fan of inline styling, especially when there's already a `.scss` file. If it's always done this way, I'd keep up with the project code "syntax". ;)

File: src/pages/[slug]/[ie]/index.tsx [hunk 4]
```diff
@@ -593,7 +592,18 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 							onClickAction={onClickAction}
 						/>
 					</div>
+					{!mediaInfo?.description && (
+						<Alert
+							className="c-Alert__margin-bottom"
+							icon={<Icon name="info" />}
+							title={''}
```
Comment (silkederudder-shd) [inline src/pages/[slug]/[ie]/index.tsx:599]: A11Y: maybe we can extend the alert component with a hideTitle option so we don't render an empty h-tag? 🤔 A temporary solution can be adding a className that hides the title using display none.

```css
.c-Alert__title--hidden {
    display: none;
}
```

**Review by silkederudder-shd — CHANGES_REQUESTED** [review]

File: src/pages/[slug]/[ie]/index.tsx (hunk 4 above)
Comment (WardVerc1) [reply src/pages/[slug]/[ie]/index.tsx]: See https://github.com/viaacode/react-components/pull/73

**Review by bertyhell — APPROVED** [review]

**Review by silkederudder-shd — APPROVED** [review]

**Pull request merged**